darasimiolaifa/epicmail
# #164702879 Fixes Project Errors

#What does this PR do?
This PR fixes project errors.

#What major activities were carried out?
The following major activities were carried out to achieve this
- handling click events for the UI
- setting the viewport height of the inbox, view-mail, and admin pages
- refactoring backend code to use model files rather than interface with the data store directly

#How do I test this bug fix?
To test this implementation, take the following steps:
- run git clone https://github.com/darasimiolaifa/epicmail.git
- cd epicmail
- run npm install
- visit localhost:3000 in your browser and navigate the web pages
- open the folder and examine the code
- run npm run test to see test results

PT Story link [#164702879](https://www.pivotaltracker.com/story/show/164702879)
